tests/test_inplace_hooks.py

```diff
@@ -89,6 +90,16 @@ def test_intree_backend_loaded_from_correct_backend_path():
     assert res == ["intree_backend_called"]
 
 
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="no importlib.metadata")
```

Comment:
You may prefer to use `pytest.importorskip('importlib.metadata')` (in the code).